### ISSUE TITLE
很喜欢这个项目  我可以加入开发吗？

### ISSUE THREAD

**Issue by zhangxianweihebei** [badarshahzad/JFX-Browser#72]
时间可能不太多，但是很感兴趣

**Comment by badarshahzad** [comment]
如果你的英语上升问题意味着你想加入这个项目。那你最受欢迎

**Comment by zhangxianweihebei** [comment]
I don’t understand what you said in Chinese.

**Comment by zhangxianweihebei** [comment]
my English is not good

**Comment by badarshahzad** [comment]
Well thank you to speak in English. Its okay. Its not our mother language. Although, if your issue meant to join our project then I said most welcome. Because I translated the chinese into english the meaning was this:

很喜欢这个项目 我可以加入开发吗？ 
I really like this project. Can I join the development?

**Comment by naeemkhan12** [comment]
you are right, browser lacks some essential features that are part of any reasonable web browser, but you can contribute to help us make it more productive and it would be much appreciated if you could comment in English to help others understand as well. Thank you.

**Comment by badarshahzad** [comment]
@zhangxianweihebei The rule of thumb is just simple made your contribution :100: . 

* Clone this repo
* Made changes in the project which should be flexible and acceptable
* Send a pull request
* After reviewing your changes, you contribution will be merge
* You will be officially added into repo as contributor :1st_place_medal: 
That's the story of a contributor. :+1:

**Comment by zhangxianweihebei** [comment]
Ok
